# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#: ../DistUpgrade/DistUpgradeController.py:1045
msgid ""
msgstr ""
"Project-Id-Version: update-manager HEAD\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2021-06-07 09:13-0700\n"
"PO-Revision-Date: 2019-06-18 17:52+0000\n"
"Last-Translator: Brian Murray <brian@ubuntu.com>\n"
"Language-Team: Nederlands <vertaling@vrijschrift.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2019-11-15 04:46+0000\n"
"X-Generator: Launchpad (build c597c3229eb023b1e626162d5947141bf7befb13)\n"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:215 ../DistUpgrade/distro.py:446
#, python-format
msgid "Server for %s"
msgstr "Server voor %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:233 ../DistUpgrade/distro.py:239
#: ../DistUpgrade/distro.py:255
msgid "Main server"
msgstr "Hoofdserver"

#: ../DistUpgrade/distro.py:259
msgid "Custom servers"
msgstr "Aangepaste servers"

#: ../DistUpgrade/DistUpgradeCache.py:182
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Pakket dat in slechte staat is verwijderen"
msgstr[1] "Pakketten die in slechte staat zijn verwijderen"

#: ../DistUpgrade/DistUpgradeCache.py:185
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Het pakket ‘%s’ verkeert in een inconsistente staat en moet opnieuw "
"geïnstalleerd worden. Er is echter geen archiefbestand gevonden voor dit "
"pakket. Wilt u het pakket verwijderen en doorgaan?"
msgstr[1] ""
"De pakketten ‘%s’ verkeren in een inconsistente staat en moeten opnieuw "
"geïnstalleerd worden. Er zijn echter geen archiefbestanden gevonden voor "
"deze pakketten. Wilt u de pakketten verwijderen en doorgaan?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:301
msgid "The server may be overloaded"
msgstr "De server is mogelijk overbelast"

#: ../DistUpgrade/DistUpgradeCache.py:414
msgid "Broken packages"
msgstr "Niet-werkende pakketten"

#: ../DistUpgrade/DistUpgradeCache.py:415
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Uw systeem bevat niet-werkende pakketten die niet hersteld kunnen worden met "
"deze software. Herstel deze eerst met synaptic of apt-get voordat u "
"verdergaat."

#: ../DistUpgrade/DistUpgradeCache.py:660
#, fuzzy, python-format
msgid "Broken packages after upgrade: %s"
msgstr "Niet-werkende pakketten"

#. the most likely problem is the 3rd party pkgs so don't address
#. foreignPkgs and devRelease being True
#: ../DistUpgrade/DistUpgradeCache.py:669
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
" "
msgstr ""
"Er heeft zich een onoplosbaar probleem voorgedaan bij het berekenen van de "
"upgrade.\n"
"\n"
" "

#: ../DistUpgrade/DistUpgradeCache.py:672
msgid ""
"This was likely caused by:\n"
" * Unofficial software packages not provided by Ubuntu\n"
"Please use the tool 'ppa-purge' from the ppa-purge \n"
"package to remove software from a Launchpad PPA and \n"
"try the upgrade again.\n"
"\n"
msgstr ""
"Dit komt waarschijnlijk door:\n"
" * Onofficiële softwarepakketten die niet door Ubuntu verstrekt worden\n"
"Gebruik het hulpmiddel ‘ppa-purge’ uit het ppa-purge-pakket \n"
"om software te verwijderen uit een Launchpad PPA en \n"
"probeer de upgrade opnieuw.\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:680
msgid ""
"This was caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
"This is most likely a transient problem, \n"
"please try again later.\n"
msgstr ""
"Dit komt waarschijnlijk door:\n"
" * het upgraden naar een pre-release-versie van Ubuntu\n"
"Dit is hoogstwaarschijnlijk een tijdelijk probleem, \n"
"probeer het later opnieuw.\n"

#: ../DistUpgrade/DistUpgradeCache.py:687
msgid "This is most likely a transient problem, please try again later."
msgstr ""
"Dit is waarschijnlijk een tijdelijk probleem.\r\n"
"Probeer het later nog eens."

#: ../DistUpgrade/DistUpgradeCache.py:690
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug ubuntu-release-upgrader-core' in a terminal. "
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:692
msgid ""
"If you want to investigate this yourself the log files in '/var/log/dist-"
"upgrade' will contain details about the upgrade. Specifically, look at 'main."
"log' and 'apt.log'."
msgstr ""
"Als u dit zelf wilt onderzoeken kunt u de logbestanden in ‘/var/log/dist-"
"upgrade’ raadplegen; deze bevatten  details over de upgrade. Kijk vooral "
"naar ‘main.log’ en ‘apt.log’."

#: ../DistUpgrade/DistUpgradeCache.py:698
msgid "Could not calculate the upgrade"
msgstr "Berekenen van de opwaardering is mislukt"

#: ../DistUpgrade/DistUpgradeCache.py:757
msgid "Error authenticating some packages"
msgstr "Fout bij het bepalen van de echtheid van sommige pakketten"

#: ../DistUpgrade/DistUpgradeCache.py:758
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Het was niet mogelijk om de echtheid van sommige pakketten vast te stellen. "
"Dit kan liggen aan een tijdelijk netwerkprobleem. U kunt het later opnieuw "
"proberen. Hieronder vindt u een lijst met pakketten waarvan de echtheid niet "
"vastgesteld is."

#: ../DistUpgrade/DistUpgradeCache.py:779
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"Het pakket ‘%s’ is gemarkeerd voor verwijdering, maar het staat op de zwarte "
"lijst voor verwijderen."

#: ../DistUpgrade/DistUpgradeCache.py:785
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Het essentiële pakket ‘%s’ is gemarkeerd voor verwijdering."

#: ../DistUpgrade/DistUpgradeCache.py:794
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr ""
"Pakket (versie ‘%s’) dat op de zwarte lijst staat proberen te installeren"

#: ../DistUpgrade/DistUpgradeCache.py:916
#, python-format
msgid "Can't install '%s'"
msgstr "Kan ‘%s’ niet installeren"

#: ../DistUpgrade/DistUpgradeCache.py:917
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""
"Het was onmogelijk om een vereist pakket te installeren. Meld deze fout a.u."
"b. met behulp van de terminalopdracht 'ubuntu-bug ubuntu-release-upgrader-"
"core'."

#: ../DistUpgrade/DistUpgradeCache.py:928
msgid "Can't guess meta-package"
msgstr "Raden van het metapakket is mislukt"

#: ../DistUpgrade/DistUpgradeCache.py:929
#, python-format
msgid ""
"Your system does not contain a %s or %s package and it was not possible to "
"detect which version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Er is geen %s- of %s-pakket op uw systeem aangetroffen en het was niet "
"mogelijk te achterhalen welke versie van Ubuntu u heeft.\n"
" Gelieve eerst één van bovengenoemde pakketten te installeren via synaptic "
"of apt-get alvorens door te gaan."

#: ../DistUpgrade/DistUpgradeController.py:112
msgid "Reading cache"
msgstr "Tijdelijke opslag inlezen"

#: ../DistUpgrade/DistUpgradeController.py:219
msgid "Unable to get exclusive lock"
msgstr "Verkrijgen van exclusieve vergrendeling is niet mogelijk"

#: ../DistUpgrade/DistUpgradeController.py:220
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Dit betekent meestal dat een andere pakketbeheerprogramma (zoals apt-get of "
"aptitude) actief is. Gelieve die toepassing eerst te sluiten."

#: ../DistUpgrade/DistUpgradeController.py:278
msgid "Upgrading over remote connection not supported"
msgstr "Opwaarderen via een externe verbinding wordt niet ondersteund"

#: ../DistUpgrade/DistUpgradeController.py:279
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"U voert de opwaardering uit via een ssh-verbinding op afstand, met een "
"bedieningsschil die dit niet ondersteunt. Probeer een opwaardering in "
"tekstmodus met 'do-release-upgrade'.\n"
"\n"
"De opwaardering wordt nu afgebroken. Probeer het a.u.b. opnieuw zonder ssh."

#: ../DistUpgrade/DistUpgradeController.py:293
msgid "Continue running under SSH?"
msgstr "Doorgaan met uitvoeren via ssh?"

#: ../DistUpgrade/DistUpgradeController.py:294
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Deze sessie lijkt onder ssh te draaien. Het wordt momenteel niet aangeraden "
"om een opwaardering via ssh uit te voeren, omdat een mislukte opwaardering "
"dan moeilijk te herstellen is.\n"
"\n"
"Als u verdergaat, zal er een extra ssh-achtergronddienst gestart worden op "
"poort '%s'.\n"
"Wilt u verdergaan?"

#: ../DistUpgrade/DistUpgradeController.py:308
msgid "Starting additional sshd"
msgstr "Bezig met starten van extra sshd"

#: ../DistUpgrade/DistUpgradeController.py:309
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Om bij fouten een hersteloperatie te vergemakkelijken, zal een extra sshd "
"gestart worden op poort ‘%s’. Indien er iets misgaat met de ssh die in "
"gebruik was, kunt u nog steeds verbinden met de extra sshd.\n"

#: ../DistUpgrade/DistUpgradeController.py:317
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Als u een firewall heeft draaien kan het zijn dat u tijdelijk deze poort "
"dient te openen. Dit wordt niet automatisch gedaan, aangezien dit gevaarlijk "
"kan zijn. U kunt de poort openen met bijv.:\n"
"‘%s’"

#: ../DistUpgrade/DistUpgradeController.py:395
#: ../DistUpgrade/DistUpgradeController.py:419
msgid "Can not upgrade"
msgstr "Opwaarderen niet mogelijk"

#: ../DistUpgrade/DistUpgradeController.py:396
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr ""
"Een opwaardering van '%s naar '%s' is niet mogelijk met dit gereedschap."

#: ../DistUpgrade/DistUpgradeController.py:420
msgid ""
"Your python3 install is corrupted. Please fix the '/usr/bin/python3' symlink."
msgstr ""
"De installatie van python3 is beschadigd. Gelieve de symlink ‘/usr/bin/"
"python3’ te repareren."

#: ../DistUpgrade/DistUpgradeController.py:449
#, python-format
msgid "Can not write to '%s'"
msgstr "Schrijven naar ‘%s’ is niet mogelijk"

#: ../DistUpgrade/DistUpgradeController.py:450
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""
"Het is niet mogelijk om naar de systeemmap ‘%s’ op uw systeem te schrijven. "
"De opwaardering kan niet verder gaan.\n"
"Zorg ervoor dat er in de systeemmap geschreven kan worden."

#: ../DistUpgrade/DistUpgradeController.py:617
#, python-format
msgid "Not for humans during development stage of release %s"
msgstr "Niet voor mensen gedurende het ontwikkelstadium van uitgave %s"

#: ../DistUpgrade/DistUpgradeController.py:717
#, python-format
msgid "disabled on upgrade to %s"
msgstr "uitgeschakeld bij opwaardering naar %s"

#: ../DistUpgrade/DistUpgradeController.py:776
msgid "No valid sources.list entry found"
msgstr "Geen geldige bronnenlijstregel gevonden"

#: ../DistUpgrade/DistUpgradeController.py:777
#, python-format
msgid ""
"While scanning your repository information no entry about %s could be "
"found.\n"
"\n"
"An upgrade might not succeed.\n"
"\n"
"Do you want to continue anyway?"
msgstr ""
"Tijdens het doorzoeken van uw pakketbroninformatie kon er geen regel worden "
"gevonden inzake %s.\n"
"\n"
"Een opwaardering zou kunnen mislukken.\n"
"\n"
"Wilt u toch doorgaan?"

#: ../DistUpgrade/DistUpgradeController.py:787
msgid "No valid mirror found"
msgstr "Geen geldige spiegelserver gevonden"

#: ../DistUpgrade/DistUpgradeController.py:788
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run an internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"Bij het doorzoeken van uw softwarebronnen werd geen spiegelserver voor de "
"opwaardering gevonden. Dit kan veroorzaakt worden door het draaien van een "
"interne spiegelserver, of doordat de spiegelserverinformatie verouderd is.\n"
"\n"
"Wilt u uw 'sources.list'-bestand toch herschrijven? Als u 'Ja' kiest zullen "
"alle '%s' in '%s' veranderd worden.\n"
"Als u 'Nee' kiest zal de opwaardering afgebroken worden."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:808
msgid "Generate default sources?"
msgstr "De standaard bronnenlijst aanmaken?"

#: ../DistUpgrade/DistUpgradeController.py:809
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Tijdens het doorzoeken van uw beschikbare softwarebronnen is er geen "
"geschikte server voor ’%s’ gevonden.\n"
"\n"
"Moeten de standaardservers voor ‘%s’ toegevoegd worden? Als u 'Nee' kiest, "
"wordt de opwaardering geannuleerd."

#: ../DistUpgrade/DistUpgradeController.py:844
#: ../DistUpgrade/DistUpgradeController.py:850
msgid "Repository information invalid"
msgstr "De informatie over de pakketbronnen is ongeldig"

#: ../DistUpgrade/DistUpgradeController.py:845
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""
"Het updaten van de softwarebron resulteerde in een ongeldig bestand. Er "
"wordt automatisch een bug-rapport aangemaakt."

#: ../DistUpgrade/DistUpgradeController.py:851
msgid ""
"Upgrading the repository information resulted in a invalid file. To report a "
"bug install apport and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"Het opwaarderen van de softwarebroninformatie resulteerde in een ongeldig "
"bestand. Om een fout te melden moet apport geïnstalleerd zijn en dient u "
"'apport-bug ubuntu-release-upgrader' uit te voeren."

#: ../DistUpgrade/DistUpgradeController.py:860
msgid "Third party sources disabled"
msgstr "Pakketbronnen van derden uitgeschakeld"

#: ../DistUpgrade/DistUpgradeController.py:861
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Enkele pakketbronnen van derden in uw sources.list zijn uitgeschakeld. U "
"kunt ze na de opwaardering weer inschakelen met het programma ‘software-"
"properties’ of met uw pakketbeheerder."

#: ../DistUpgrade/DistUpgradeController.py:903
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Pakket in inconsistente staat"
msgstr[1] "Pakketten in inconsistente staat"

#: ../DistUpgrade/DistUpgradeController.py:906
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Het pakket ‘%s’ verkeert in een inconsistente staat en moet opnieuw "
"geïnstalleerd worden. Er zijn echter geen archiefbestanden gevonden voor "
"deze pakketten. Gelieve het pakket handmatig te herinstalleren of te "
"verwijderen uit het systeem."
msgstr[1] ""
"De pakketten ‘%s’ bevinden zich in een inconsistente toestand en moeten "
"opnieuw geïnstalleerd worden, maar er is geen archief gevonden dat de "
"pakketten bevat. U moet de pakketten zelf installeren of u moet ze "
"verwijderen uit het systeem."

#: ../DistUpgrade/DistUpgradeController.py:969
msgid "Error during update"
msgstr "Fout tijdens het bijwerken"

#: ../DistUpgrade/DistUpgradeController.py:970
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Tijdens de update heeft zich een probleem voorgedaan. De oorzaak hiervan is "
"meestal een netwerkprobleem. Gelieve uw netwerkverbinding te controleren en "
"probeer het dan opnieuw."

#: ../DistUpgrade/DistUpgradeController.py:1013
#: ../DistUpgrade/DistUpgradeController.py:1018
msgid "EFI System Partition (ESP) not usable"
msgstr "EFI-systeempartitie (ESP) niet bruikbaar"

#: ../DistUpgrade/DistUpgradeController.py:1014
msgid ""
"Your EFI System Partition (ESP) is not mounted at /boot/efi. Please ensure "
"that it is properly configured and try again."
msgstr ""
"Uw EFI-systeempartitie (ESP) is niet gekoppeld aan /boot/efi. Zorg ervoor "
"dat de configuratie correct is en probeer het opnieuw."

#: ../DistUpgrade/DistUpgradeController.py:1019
msgid ""
"The EFI System Partition (ESP) mounted at /boot/efi is not writable. Please "
"mount this partition read-write and try again."
msgstr ""
"De EFI-systeempartitie (ESP) gekoppeld aan /boot/efi is niet beschrijfbaar. "
"Koppel deze partitie met lees- en schrijfrechten aan en probeer het opnieuw."

#: ../DistUpgrade/DistUpgradeController.py:1026
msgid "Not enough free disk space"
msgstr "Niet genoeg vrije schijfruimte"

#. TRANSLATORS: you can change the order of the sentence,
#. make sure to keep all {str_*} string untranslated.
#: ../DistUpgrade/DistUpgradeController.py:1029
#, python-brace-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of {str_total} free space "
"on disk '{str_dir}'. Please free at least an additional {str_needed} of disk "
"space on '{str_dir}'. {str_remedy}"
msgstr ""
"De upgrade is afgebroken. De upgrade heeft in totaal {str_total} vrije "
"ruimte op schijf ‘{str_dir}’ nodig. Maak ten minste {str_needed} extra "
"schijfruimte vrij op ‘{str_dir}’. {str_remedy}"

#. specific ways to resolve lack of free space
#: ../DistUpgrade/DistUpgradeController.py:1034
msgid ""
"Remove temporary packages of former installations using 'sudo apt clean'."
msgstr ""
"Verwijder tijdelijke pakketten van vorige installaties met de opdracht ‘sudo "
"apt clean’."

#: ../DistUpgrade/DistUpgradeController.py:1036
msgid ""
"You can remove old kernels using 'sudo apt autoremove' and you could also "
"set COMPRESS=xz in /etc/initramfs-tools/initramfs.conf to reduce the size of "
"your initramfs."
msgstr ""
"U kunt oude kernels verwijderen via de opdracht ‘sudo apt autoremove’. Ook "
"kunt u in /etc/initramfs-tools/initramfs.conf COMPRESS=xz instellen om de "
"grootte van uw initramfs te verkleinen."

#: ../DistUpgrade/DistUpgradeController.py:1041
msgid ""
"Empty your trash and remove temporary packages of former installations using "
"'sudo apt-get clean'."
msgstr ""
"Maak uw prullenbak leeg en verwijder tijdelijke pakketten van vorige "
"instalatie via de opdracht ‘sudo apt-get clean’."

#: ../DistUpgrade/DistUpgradeController.py:1044
msgid "Reboot to clean up files in /tmp."
msgstr "Herstart om bestanden in /tmp te wissen."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:1083
#: ../DistUpgrade/DistUpgradeController.py:1925
#: ../DistUpgrade/DistUpgradeController.py:1977
msgid "Calculating the changes"
msgstr "Wijzigingen worden berekend"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:1127
msgid "Do you want to start the upgrade?"
msgstr "Wilt u beginnen met het opwaarderen?"

#: ../DistUpgrade/DistUpgradeController.py:1146
#, python-format
msgid ""
"Livepatch security updates are not available for Ubuntu %s. If you upgrade, "
"Livepatch will turn off."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1218
msgid "Upgrade canceled"
msgstr "Opwaardering geannuleerd"

#: ../DistUpgrade/DistUpgradeController.py:1219
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"De opwaardering zal nu worden geannuleerd en de originele systeemstatus "
"wordt hersteld. U kunt de opwaardering op een later tijdstip hervatten."

#: ../DistUpgrade/DistUpgradeController.py:1225
#: ../DistUpgrade/DistUpgradeController.py:1370
msgid "Could not download the upgrades"
msgstr "Kon de opwaarderingsbestanden niet binnenhalen"

#: ../DistUpgrade/DistUpgradeController.py:1226
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"De opwaardering is afgebroken. Controleer a.u.b. uw internetverbinding of "
"installatiemedia en probeer het opnieuw. Alle bestanden die tot nu toe "
"binnengehaald zijn zullen bewaard blijven."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1319
#: ../DistUpgrade/DistUpgradeController.py:1358
#: ../DistUpgrade/DistUpgradeController.py:1474
msgid "Error during commit"
msgstr "Fout bij het toepassen"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1321
#: ../DistUpgrade/DistUpgradeController.py:1360
#: ../DistUpgrade/DistUpgradeController.py:1519
msgid "Restoring original system state"
msgstr "De oorspronkelijke toestand wordt hersteld"

#: ../DistUpgrade/DistUpgradeController.py:1322
#: ../DistUpgrade/DistUpgradeController.py:1337
#: ../DistUpgrade/DistUpgradeController.py:1361
msgid "Could not install the upgrades"
msgstr "Installatie van de opwaarderingsbestanden is mislukt"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1327
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"De opwaardering is afgebroken. Uw systeem kan zich in een onbruikbare staat "
"bevinden. Er zal nu een herstelprocedure uitgevoerd worden (dpkg --configure "
"-a)."

#: ../DistUpgrade/DistUpgradeController.py:1332
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at http://bugs.launchpad.net/ubuntu/"
"+source/ubuntu-release-upgrader/+filebug and attach the files in /var/log/"
"dist-upgrade/ to the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Meld deze fout a.u.b. in een webbrowser op http://bugs.launchpad.net/ubuntu/"
"+source/ubuntu-release-upgrader/+filebug and voeg de bestanden in /var/log/"
"dist-upgrade/ toe aan het foutrapport.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1371
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"De opwaardering is afgebroken. Controleer a.u.b. uw internetverbinding of "
"installatiemedia en probeer het opnieuw. "

#: ../DistUpgrade/DistUpgradeController.py:1399
msgid "Searching for obsolete software"
msgstr "Zoeken naar verouderde software"

#: ../DistUpgrade/DistUpgradeController.py:1462
msgid "Remove obsolete packages?"
msgstr "Verouderde pakketten verwijderen?"

#: ../DistUpgrade/DistUpgradeController.py:1463
#: ../data/gtkbuilder/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "_Behouden"

#: ../DistUpgrade/DistUpgradeController.py:1463
msgid "_Remove"
msgstr "_Verwijderen"

#: ../DistUpgrade/DistUpgradeController.py:1475
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Tijdens het opruimen heeft zich een probleem voorgedaan. Zie onderstaand "
"bericht voor meer informatie. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1557
msgid "Required depends is not installed"
msgstr "De vereiste afhankelijkheden zijn niet geïnstalleerd"

#: ../DistUpgrade/DistUpgradeController.py:1558
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Het vereiste pakket ‘%s’ is niet geïnstalleerd. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1791
#: ../DistUpgrade/DistUpgradeController.py:1875
msgid "Checking package manager"
msgstr "Pakketbeheerder controleren"

#: ../DistUpgrade/DistUpgradeController.py:1797
#: ../DistUpgrade/DistUpgradeController.py:1803
msgid "Preparing the upgrade failed"
msgstr "Voorbereiden van de opwaardering is mislukt"

#: ../DistUpgrade/DistUpgradeController.py:1798
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""
"Het voorbereiden van het systeem op de opwaardering is mislukt, dus wordt er "
"een foutrapportageproces gestart."

#: ../DistUpgrade/DistUpgradeController.py:1804
msgid ""
"Preparing the system for the upgrade failed. To report a bug install apport "
"and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"Voorbereiden van het systeem voor het opwaarderen is mislukt. Om een fout te "
"melden moet apport geïnstalleerd zijn en dient u 'apport-bug ubuntu-release-"
"upgrader' uit te voeren."

#: ../DistUpgrade/DistUpgradeController.py:1821
#: ../DistUpgrade/DistUpgradeController.py:1831
msgid "Getting upgrade prerequisites failed"
msgstr ""
"Het verkrijgen van de vooraf benodigde zaken voor de opwaardering is mislukt"

#: ../DistUpgrade/DistUpgradeController.py:1822
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""
"Het systeem kon de vooraf benodigde zaken voor de opwaardering niet "
"verkrijgen. De opwaardering wordt nu afgebroken en het systeem wordt in de "
"originele staat hersteld.\n"
"\n"
"Daarnaast wordt er een proces gestart voor de foutmelding."

#: ../DistUpgrade/DistUpgradeController.py:1832
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"To report a bug install apport and then execute 'apport-bug ubuntu-release-"
"upgrader'."
msgstr ""
"Het systeem kon de vereisten voor de opwaardering niet verkrijgen. De "
"opwaardering zal nu worden afgebroken en de originele systeemstatus zal "
"hersteld worden.\n"
"\n"
"Om een fout te melden moet apport geïnstalleerd zijn en dient u 'apport-bug "
"ubuntu-release-upgrader' uit te voeren."

#: ../DistUpgrade/DistUpgradeController.py:1866
msgid "Updating repository information"
msgstr "Updaten van de informatie over de pakketbronnen"

#: ../DistUpgrade/DistUpgradeController.py:1902
msgid "Invalid package information"
msgstr "Ongeldige pakketinformatie"

#: ../DistUpgrade/DistUpgradeController.py:1903
#, python-format
msgid ""
"After updating your package information, the essential package '%s' could "
"not be located. This may be because you have no official mirrors listed in "
"your software sources, or because of excessive load on the mirror you are "
"using. See /etc/apt/sources.list for the current list of configured software "
"sources.\n"
"In the case of an overloaded mirror, you may want to try the upgrade again "
"later."
msgstr ""
"Na het bijwerken van uw pakketinformatie, kon het essentiële pakket '%s' "
"niet worden gevonden. Dit zou kunnen komen doordat er geen officiële "
"spiegelservers in uw pakketbronnenlijst staan, of door bovenmatige belasting "
"van de spiegelserver die u gebruikt. Zie /etc/apt/sources.list voor de "
"huidige lijst van ingestelde pakketbronnen.\n"
"In geval van een overbelaste spiegelserver, zou u op een later tijdstip de "
"opwaardering wederom kunnen proberen."

#: ../DistUpgrade/DistUpgradeController.py:1932
#: ../DistUpgrade/DistUpgradeController.py:2038
msgid "Fetching"
msgstr "Ophalen"

#: ../DistUpgrade/DistUpgradeController.py:1939
#: ../DistUpgrade/DistUpgradeController.py:1962
#: ../DistUpgrade/DistUpgradeController.py:1992
#: ../DistUpgrade/DistUpgradeController.py:2042
msgid "Upgrading"
msgstr "Bezig met opwaarderen"

#: ../DistUpgrade/DistUpgradeController.py:1941
msgid "Upgrade infeasible"
msgstr "Upgrade onuitvoerbaar"

#: ../DistUpgrade/DistUpgradeController.py:1942
msgid ""
"The upgrade could not be completed, there were errors during the upgrade "
"process."
msgstr ""
"De upgrade kon niet worden voltooid; er deden zich fouten voor tijdens het "
"upgradeproces."

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1965
#: ../DistUpgrade/DistUpgradeController.py:1981
msgid "Upgrade incomplete"
msgstr "Upgrade onvolledig"

#: ../DistUpgrade/DistUpgradeController.py:1966
#: ../DistUpgrade/DistUpgradeController.py:1982
msgid ""
"The upgrade has partially completed but there were errors during the upgrade "
"process."
msgstr ""
"De upgrade is deels voltooid, maar er deden zich fouten voor tijdens het "
"upgradeproces."

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1997
#: ../DistUpgrade/DistUpgradeController.py:2044
#: ../DistUpgrade/DistUpgradeController.py:2050
#: ../DistUpgrade/DistUpgradeController.py:2061
msgid "Upgrade complete"
msgstr "Opwaardering is voltooid"

#: ../DistUpgrade/DistUpgradeController.py:1998
#: ../DistUpgrade/DistUpgradeController.py:2045
#: ../DistUpgrade/DistUpgradeController.py:2051
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr ""
"De opwaardering is voltooid, maar er hebben zich fouten voorgedaan tijdens "
"het opwaardeerproces."

#: ../DistUpgrade/DistUpgradeController.py:2013
msgid "System upgrade is complete."
msgstr "Systeemopwaardering is voltooid."

#: ../DistUpgrade/DistUpgradeController.py:2062
msgid "The partial upgrade was completed."
msgstr "De gedeeltelijke opwaardering werd voltooid."

#: ../DistUpgrade/DistUpgradeController.py:2091
msgid "Lock screen disabled"
msgstr "Schermvergrendeling uitgeschakeld"

#: ../DistUpgrade/DistUpgradeController.py:2092
msgid ""
"Your lock screen has been disabled and will remain disabled until you reboot."
msgstr ""
"De schermvergrendeling is uitgeschakeld en zal uitgeschakeld blijven totdat "
"u opnieuw start."

#: ../DistUpgrade/DistUpgradeFetcher.py:107
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:170
msgid "Could not find the release notes"
msgstr "Kon de versie-informatie niet vinden"

#: ../DistUpgrade/DistUpgradeFetcher.py:108
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:171
msgid "The server may be overloaded. "
msgstr "De server is waarschijnlijk overbelast. "

#: ../DistUpgrade/DistUpgradeFetcher.py:120
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:175
msgid "Could not download the release notes"
msgstr "Kon de versie-informatie niet downloaden"

#: ../DistUpgrade/DistUpgradeFetcher.py:121
#: ../DistUpgrade/DistUpgradeFetcherCore.py:75
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:176
msgid "Please check your internet connection."
msgstr "Controleer uw internetverbinding."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:70
#, fuzzy
msgid "Could not find the release announcement"
msgstr "Kon de versie-informatie niet vinden"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:71
#, fuzzy
msgid "The server may be overloaded."
msgstr "De server is waarschijnlijk overbelast. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:74
#, fuzzy
msgid "Could not download the release announcement"
msgstr "Kon de versie-informatie niet downloaden"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:80
#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
#: ../DistUpgrade/DistUpgradeViewText.py:261
msgid "Continue [yN] "
msgstr "_Doorgaan [jN] "

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeFetcherCore.py:82
#: ../DistUpgrade/DistUpgradeViewText.py:219
#: ../DistUpgrade/DistUpgradeViewText.py:264
msgid "y"
msgstr "j"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:99
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr "'%(file)s' tegen  '%(signature)s' verifiëren "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:125
#, python-format
msgid "extracting '%s'"
msgstr "‘%s’ wordt uitgepakt"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:146
#: ../DistUpgrade/DistUpgradeFetcherCore.py:147
msgid "Could not run the upgrade tool"
msgstr "Uitvoering van het opwaardeerprogramma is mislukt"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:148
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug ubuntu-release-upgrader-core'."
msgstr ""
"Dit is waarschijnlijk een fout in het opwaardeerprogramma. Meld deze fout a."
"u.b. met behulp van de terminalopdracht 'ubuntu-bug ubuntu-release-upgrader-"
"core'."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:225
msgid "Upgrade tool signature"
msgstr "Handtekening opwaardeerprogramma"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:232
msgid "Upgrade tool"
msgstr "Opwaardeerprogramma"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:266
msgid "Failed to fetch"
msgstr "Ophalen is mislukt"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:267
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Het ophalen van de opwaardering is mislukt. Er is mogelijk een "
"netwerkprobleem. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:271
msgid "Authentication failed"
msgstr "Echtheidscontrole is mislukt"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:272
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"De echtheidscontrole van de opwaardering is mislukt. Er is mogelijk een "
"probleem met het netwerk of met de server. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:277
msgid "Failed to extract"
msgstr "Uitpakken is mislukt"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:278
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Het uitpakken van de opwaardering is mislukt. Er is mogelijk een probleem "
"met het netwerk of met de server. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:283
msgid "Verification failed"
msgstr "Verificatie is mislukt"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:284
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Het verifiëren van de opwaardering is mislukt. Er is mogelijk een probleem "
"met het netwerk of met de server. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:298
#: ../DistUpgrade/DistUpgradeFetcherCore.py:304
msgid "Can not run the upgrade"
msgstr "Kan de opwaardering niet uitvoeren"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:299
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Dit wordt meestal veroorzaakt door een systeem waar /tmp met noexec is "
"aangekoppeld. Koppel opnieuw aan zonder noexec en voer de opwaardering weer "
"uit."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:305
#, python-format
msgid "The error message is '%s'."
msgstr "De foutmelding is ‘%s’."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:151
msgid "&Upgrade"
msgstr "&Opwaarderen"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:154
#: ../DistUpgrade/DistUpgradeViewKDE.py:715
msgid "&Cancel"
msgstr "&Annuleren"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:156
#: ../data/gtkbuilder/ReleaseNotes.ui.h:1
msgid "Release Notes"
msgstr "Versie-informatie"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:191
msgid "Upgrade"
msgstr "Opwaarderen"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:207
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:222
msgid "Downloading additional package files..."
msgstr "Aanvullende pakketbestanden downloaden…"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:224
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Bestand %s van %s met %sB/s"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:228
#, python-format
msgid "File %s of %s"
msgstr "Bestand %s van %s"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:235
#: ../DistUpgrade/DistUpgradeViewGtk3.py:125
#: ../DistUpgrade/DistUpgradeViewKDE.py:302
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Plaats ‘%s’ in station ‘%s’"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:237
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:242
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:243
#: ../DistUpgrade/DistUpgradeViewKDE.py:303
msgid "Media Change"
msgstr "Medium wisselen"

#: ../DistUpgrade/DistUpgradeQuirks.py:231
msgid "Your graphics hardware may not be fully supported in Ubuntu 14.04."
msgstr ""
"Mogelijk wordt uw grafische kaart niet volledig ondersteund in Ubuntu 14.04."

#: ../DistUpgrade/DistUpgradeQuirks.py:233
msgid ""
"Running the 'unity' desktop environment is not fully supported by your "
"graphics hardware. You will maybe end up in a very slow environment after "
"the upgrade. Our advice is to keep the LTS version for now. For more "
"information see https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D Do you still want to continue with the "
"upgrade?"
msgstr ""
"Het draaien van de 'Unity'-werkomgeving wordt niet volledig ondersteund door "
"uw videokaart. U zou mogelijk een erg langzaam systeem kunnen krijgen na de "
"opwaardering. Ons advies is, om voorlopig bij de LTS-versie te blijven. Meer "
"informatie vindt u op https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D  Wilt u toch doorgaan met de opwaardering?"

#: ../DistUpgrade/DistUpgradeQuirks.py:257
msgid "Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr ""
"Uw grafische hardware wordt misschien niet volledig ondersteund in Ubuntu "
"12.04 LTS."

#: ../DistUpgrade/DistUpgradeQuirks.py:259
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""
"De ondersteuning in Ubuntu 12.04 LTS voor uw Intel-videokaart is beperkt en "
"er kunnen problemen optreden na de opwaardering. Voor meer informatie zie "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Wilt u toch "
"verder gaan met de opwaardering?"

#: ../DistUpgrade/DistUpgradeQuirks.py:279
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Opwaarderen zou de bureaubladeffecten kunnen verminderen, alsook de "
"prestaties in spellen en andere grafisch intensieve programma's."

#: ../DistUpgrade/DistUpgradeQuirks.py:283
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Deze computer gebruikt momenteel het grafisch stuurprogramma 'fglrx' van "
"AMD. Voor Ubuntu 10.04 LTS is geen stuurprogramma beschikbaar dat met uw "
"hardware werkt.\n"
"Wilt u doorgaan?"

#: ../DistUpgrade/DistUpgradeQuirks.py:322
msgid "Sorry, no more upgrades for this system"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:323
#, python-format
msgid ""
"There will not be any further Ubuntu releases for this system's 'i386' "
"architecture.\n"
"\n"
"Updates for Ubuntu %s will continue until %s."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:338
msgid "No ARMv6 CPU"
msgstr "Geen ARMv6-CPU"

#: ../DistUpgrade/DistUpgradeQuirks.py:339
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Uw systeem gebruikt een ARM-CPU ouder dan de ARMv6-architectuur. Alle "
"pakketten zijn gecompileerd voor ARMv6 als de minimale architectuur. Het is "
"met uw apparatuur niet mogelijk om uw systeem op te waarderen naar een "
"nieuwe versie van Ubuntu."

#: ../DistUpgrade/DistUpgradeQuirks.py:359
msgid ""
"Sorry, this storage driver is not supported in kernels for newer releases"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:361
#, python-format
msgid ""
"There will not be any further Ubuntu releases that provide kernel support "
"for the aufs storage driver.\n"
"\n"
"Please ensure that none of your containers are using the aufs storage "
"driver, remove the directory %s and try again."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:383
#, fuzzy
msgid "An upgrade is not possible at this time"
msgstr "Uitgave-opwaardering momenteel niet mogelijk"

#: ../DistUpgrade/DistUpgradeQuirks.py:384
msgid ""
"Due to a bug in shim, LP: #1928434, upgrades are not currently safe for your "
"hardware.\n"
"\n"
"Once that bug has been resolved you will be able to upgrade to the next "
"release of Ubuntu."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:402
msgid "No init available"
msgstr "Geen init beschikbaar"

#: ../DistUpgrade/DistUpgradeQuirks.py:403
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Het lijkt erop dat uw systeem gevirtualiseerd is zonder een init-daemon, "
"bijvoorbeeld Linux-VServer. Ubuntu 10.04 LTS kan niet functioneren in dit "
"type virtualisatie. U zult eerst de instellingen van uw virtuele machine "
"moeten aanpassen.\n"
"\n"
"Weet u zeker dat u wilt doorgaan?"

#: ../DistUpgrade/DistUpgradeQuirks.py:444
msgid "The Robot Operating System (ROS) is installed"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:445
msgid ""
"It appears that ROS is currently installed. Each ROS release is very strict "
"about the versions of Ubuntu it supports, and Ubuntu upgrades can fail if "
"that guidance isn't followed. Before continuing, please either uninstall "
"ROS, or ensure the ROS release you have installed supports the version of "
"Ubuntu to which you're upgrading.\n"
"\n"
"For ROS 1 releases, refer to REP 3:\n"
"https://www.ros.org/reps/rep-0003.html\n"
"\n"
"For ROS 2 releases, refer to REP 2000:\n"
"https://www.ros.org/reps/rep-2000.html\n"
"\n"
"Are you sure you want to continue?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:576
#, fuzzy
msgid "Connection to the Snap Store failed"
msgstr "Verbinding met Snap Store is mislukt"

#: ../DistUpgrade/DistUpgradeQuirks.py:577
msgid ""
"You have the package lxd installed but your system is unable to reach the "
"Snap Store. lxd is now provided via a snap and the release upgrade will fail "
"if snapd is not functional. Please make sure you're connected to the "
"Internet and update any firewall or proxy settings as needed so that you can "
"reach api.snapcraft.io. If you are an enterprise with a firewall setup you "
"may want to configure a Snap Store proxy."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:592
msgid "Connection to Snap Store failed"
msgstr "Verbinding met Snap Store is mislukt"

#: ../DistUpgrade/DistUpgradeQuirks.py:593
msgid ""
"Your system does not have a connection to the Snap Store. For the best "
"upgrade experience make sure that your system can connect to api.snapcraft."
"io.\n"
"Do you still want to continue with the upgrade?"
msgstr ""
"Uw systeem heeft geen verbinding met Snap Store. Om de beste upgrade te "
"krijgen dient u ervoor te zorgen dat uw systeem verbinding kan maken met api."
"snapcraft.io.\n"
"Wilt u nog doorgaan met de upgrade?"

#: ../DistUpgrade/DistUpgradeQuirks.py:602
msgid "Outdated snapd package"
msgstr "Verouderd snapd-pakket"

#: ../DistUpgrade/DistUpgradeQuirks.py:603
msgid ""
"Your system does not have the latest version of snapd. Please update the "
"version of snapd on your system to improve the upgrade experience.\n"
"Do you still want to continue with the upgrade?"
msgstr ""
"Uw systeem beschikt niet over de nieuwste versie van snapd. Gelieve de "
"versie van snapd bij te werken op uw systeem om een betere upgrade te "
"krijgen.\n"
"Wilt u nog doorgaan met de upgrade?"

#. now perform direct API calls to the store, requesting size
#. information for each of the snaps needing installation
#: ../DistUpgrade/DistUpgradeQuirks.py:626
msgid "Calculating snap size requirements"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:657
msgid "Processing snap replacements"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:663
#, python-format
msgid "refreshing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:667
#, python-format
msgid "removing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:670
#, python-format
msgid "installing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:699
msgid "PAE not enabled"
msgstr "PAE is niet ingeschakeld"

#: ../DistUpgrade/DistUpgradeQuirks.py:700
msgid ""
"Your system uses a CPU that does not have PAE enabled. Ubuntu only supports "
"non-PAE systems up to Ubuntu 12.04. To upgrade to a later version of Ubuntu, "
"you must enable PAE (if this is possible) see:\n"
"http://help.ubuntu.com/community/EnablingPAE"
msgstr ""
"Uw systeem gebruikt een CPU waarbij PAE niet is ingeschakeld. Ubuntu "
"ondersteunt alleen niet-PAE-systemen tot en met Ubuntu 12.04. Om op te "
"waarderen naar een nieuwere versie van Ubuntu dient u PAE (als dat mogelijk "
"is) in te schakelen; zie:\n"
"http://help.ubuntu.com/community/EnablingPAE"

#: ../DistUpgrade/DistUpgradeQuirks.py:1072
msgid "Checking for installed snaps"
msgstr "Op geïnstalleerde snaps controleren"

#: ../DistUpgrade/DistUpgradeMain.py:62
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Gebruik een bedieningsschil. Momenteel beschikbaar: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:65
msgid "*DEPRECATED* this option will be ignored"
msgstr "*DEPRECATED* deze optie zal genegeerd worden"

#: ../DistUpgrade/DistUpgradeMain.py:68
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr ""
"Voer een gedeeltelijke opwaardering uit (sources.list wordt niet herschreven)"

#: ../DistUpgrade/DistUpgradeMain.py:71
msgid "Disable GNU screen support"
msgstr "GNU-schermondersteuning uitschakelen"

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid "Set datadir"
msgstr "Datamap instellen"

#: ../DistUpgrade/DistUpgradeMain.py:76
msgid "Upgrade to the development release"
msgstr "Opwaardering naar de ontwikkelversie"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:146
#: ../DistUpgrade/DistUpgradeViewKDE.py:316
msgid "Fetching is complete"
msgstr "Het ophalen is voltooid"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:161
#: ../DistUpgrade/DistUpgradeViewKDE.py:332
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Ophalen bestand %li van %li met %sB/s"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:163
#: ../DistUpgrade/DistUpgradeViewGtk3.py:325
#: ../DistUpgrade/DistUpgradeViewKDE.py:333
#: ../DistUpgrade/DistUpgradeViewKDE.py:483
#, python-format
msgid "About %s remaining"
msgstr "Ongeveer %s resterend"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:166
#: ../DistUpgrade/DistUpgradeViewKDE.py:335
#, python-format
msgid "Fetching file %li of %li"
msgstr "Bestand %li van %li wordt opgehaald"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk3.py:198
#: ../DistUpgrade/DistUpgradeViewKDE.py:373
msgid "Applying changes"
msgstr "Wijzigingen worden doorgevoerd"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk3.py:224
#: ../DistUpgrade/DistUpgradeViewKDE.py:386
msgid "dependency problems - leaving unconfigured"
msgstr "vereistenproblemen - blijft niet ingesteld"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:229
#: ../DistUpgrade/DistUpgradeViewKDE.py:388
#, python-format
msgid "Could not install '%s'"
msgstr "Kon ‘%s’ niet installeren"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:230
#: ../DistUpgrade/DistUpgradeViewKDE.py:389
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"De opwaardering zal doorgaan, maar het pakket '%s' zal mogelijk niet werken. "
"Overweeg a.u.b. om er een foutrapport over in te dienen."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk3.py:247
#: ../DistUpgrade/DistUpgradeViewKDE.py:411
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Het aangepaste configuratiebestand vervangen?\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:248
#: ../DistUpgrade/DistUpgradeViewKDE.py:412
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Wanneer u besluit dit configuratiebestand te vervangen door een nieuwere "
"versie zullen al uw gemaakte wijzigingen verloren gaan."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:268
#: ../DistUpgrade/DistUpgradeViewKDE.py:435
msgid "The 'diff' command was not found"
msgstr "De opdracht 'diff' is niet gevonden"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:535
#: ../DistUpgrade/DistUpgradeViewText.py:140
msgid "A fatal error occurred"
msgstr "Er is een ernstige fout opgetreden"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:536
msgid ""
"Please report this as a bug (if you haven't already) and include the files /"
"var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Rapporteer dit a.u.b. als een fout (als u dat al niet gedaan heeft) en voeg "
"de bestanden /var/log/dist-upgrade/main.log en /var/log/dist-upgrade/apt.log "
"bij uw melding. De opwaardering is afgebroken.\n"
"Uw oorspronkelijke sources.list is opgeslagen in /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:566
msgid "Ctrl-c pressed"
msgstr "Ctrl-c ingedrukt"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:567
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Dit zal de taak onderbreken waardoor uw systeem mogelijk niet meer correct "
"werkt. Weet u zeker dat u dit wilt doen?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk3.py:706
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"Sluit alle openstaande toepassingen en documenten om dataverlies te "
"voorkomen."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:720
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Niet langer ondersteund door Canonical (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:721
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Downgrade (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:722
#, python-format
msgid "Remove (%s)"
msgstr "Verwijder (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:723
#, python-format
msgid "No longer needed (%s)"
msgstr "Niet meer nodig (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:724
#, python-format
msgid "Install (%s)"
msgstr "Installeer (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:725
#, python-format
msgid "Upgrade (%s)"
msgstr "Opwaarderen (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:773
msgid "Cancel"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:774
#: ../DistUpgrade/DistUpgradeViewKDE.py:939
msgid "Continue"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewKDE.py:447
msgid "Show Difference >>>"
msgstr "Verschillen weergeven >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:450
msgid "<<< Hide Difference"
msgstr "<<< Verschillen verbergen"

#: ../DistUpgrade/DistUpgradeViewKDE.py:701
msgid "Error"
msgstr "Foutmelding"

#: ../DistUpgrade/DistUpgradeViewKDE.py:719
#: ../DistUpgrade/DistUpgradeViewKDE.py:960
msgid "&Close"
msgstr "Sl&uiten"

#: ../DistUpgrade/DistUpgradeViewKDE.py:753
msgid "Show Terminal >>>"
msgstr "Terminalvenster weergeven >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:756
msgid "<<< Hide Terminal"
msgstr "<<< Terminalvenster verbergen"

#: ../DistUpgrade/DistUpgradeViewKDE.py:829
msgid "Information"
msgstr "Informatie"

#: ../DistUpgrade/DistUpgradeViewKDE.py:877
msgid "&Start Upgrade"
msgstr "&Opwaarderen starten"

#: ../DistUpgrade/DistUpgradeViewKDE.py:879
#: ../DistUpgrade/DistUpgradeViewKDE.py:923
#: ../DistUpgrade/DistUpgradeViewKDE.py:926
#: ../data/gtkbuilder/DistUpgrade.ui.h:7
msgid "Details"
msgstr "Details"

#: ../DistUpgrade/DistUpgradeViewKDE.py:901
#, python-format
msgid "No longer supported %s"
msgstr "Niet meer ondersteund %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:903
#, python-format
msgid "Remove %s"
msgstr "%s verwijderen"

#: ../DistUpgrade/DistUpgradeViewKDE.py:905
#: ../DistUpgrade/DistUpgradeViewText.py:239
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "%s verwijderen (werd automatisch geïnstalleerd)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:907
#, python-format
msgid "Install %s"
msgstr "%s installeren"

#: ../DistUpgrade/DistUpgradeViewKDE.py:909
#, python-format
msgid "Upgrade %s"
msgstr "%s opwaarderen"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
#: ../DistUpgrade/DistUpgradeViewText.py:292
msgid "Restart required"
msgstr "Herstart vereist"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Herstart het systeem om de opwaardering te voltooien</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:959
#: ../data/gtkbuilder/DistUpgrade.ui.h:14
msgid "_Restart Now"
msgstr "_Nu herstarten"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:977
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>De opwaardering annuleren?</big></b>\n"
"\n"
"Het systeem kan onbruikbaar zijn als u de opwaardering annuleert. U wordt "
"met nadruk geadviseerd om met de opwaardering door te gaan."

#: ../DistUpgrade/DistUpgradeViewKDE.py:981
msgid "Cancel Upgrade?"
msgstr "Opwaardering annuleren?"

#: ../DistUpgrade/DistUpgradeView.py:82
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li dag"
msgstr[1] "%li dagen"

#: ../DistUpgrade/DistUpgradeView.py:84
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li uur"
msgstr[1] "%li uur"

#: ../DistUpgrade/DistUpgradeView.py:86
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li minuut"
msgstr[1] "%li minuten"

#: ../DistUpgrade/DistUpgradeView.py:87
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li seconde"
msgstr[1] "%li seconden"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:103
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:121
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:172
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Deze download zal ongeveer %s duren met een 1 Mbit DSL-verbinding en "
"ongeveer %s met een 56k-modem."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:176
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Deze download duurt met uw verbinding ongeveer %s. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:270 ../data/gtkbuilder/DistUpgrade.ui.h:22
msgid "Preparing to upgrade"
msgstr "Opwaardering aan het voorbereiden"

#: ../DistUpgrade/DistUpgradeView.py:271
msgid "Getting new software channels"
msgstr "Nieuwe softwarekanalen worden opgehaald"

#: ../DistUpgrade/DistUpgradeView.py:272 ../data/gtkbuilder/DistUpgrade.ui.h:24
msgid "Getting new packages"
msgstr "Ophalen van nieuwe pakketten"

#: ../DistUpgrade/DistUpgradeView.py:273 ../data/gtkbuilder/DistUpgrade.ui.h:27
msgid "Installing the upgrades"
msgstr "Bezig met installeren van de opwaardeerbestanden"

#: ../DistUpgrade/DistUpgradeView.py:274 ../data/gtkbuilder/DistUpgrade.ui.h:26
msgid "Cleaning up"
msgstr "Opruimen"

#: ../DistUpgrade/DistUpgradeView.py:363
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d geïnstalleerd pakket wordt niet meer door Canonical ondersteund. "
"Updates kunnen door de gemeenschap geleverd worden."
msgstr[1] ""
"%(amount)d geïnstalleerde pakketten worden niet meer door Canonical "
"ondersteund. Updates kunnen door de gemeenschap geleverd worden."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:372
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d pakket zal verwijderd worden."
msgstr[1] "%d pakketten zullen verwijderd worden."

#: ../DistUpgrade/DistUpgradeView.py:377
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d nieuw pakket zal geïnstalleerd worden."
msgstr[1] "%d nieuwe pakketten zullen geïnstalleerd worden."

#: ../DistUpgrade/DistUpgradeView.py:383
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d pakket zal opgewaardeerd worden."
msgstr[1] "%d pakketten zullen opgewaardeerd worden."

#: ../DistUpgrade/DistUpgradeView.py:391
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"U moet in totaal %s downloaden. "

#: ../DistUpgrade/DistUpgradeView.py:396
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""
"Het installeren van een opwaardering kan enkele uren in beslag nemen. Nadat "
"het ophalen voltooid is kan het proces niet meer afgebroken worden."

#: ../DistUpgrade/DistUpgradeView.py:400
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""
"Het ophalen en installeren van de pakketten kan meerdere uren duren. Wanneer "
"het ophalen van de pakketten voltooid is kan het proces niet meer afgebroken "
"worden."

#: ../DistUpgrade/DistUpgradeView.py:405
msgid "Removing the packages can take several hours. "
msgstr "Het verwijderen van de pakketten kan enkele uren in beslag nemen. "

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:410
msgid "The software on this computer is up to date."
msgstr "De software op deze computer is actueel."

#: ../DistUpgrade/DistUpgradeView.py:411
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Er zijn geen opwaarderingen beschikbaar voor uw systeem. De opwaardering "
"wordt nu geannuleerd."

#: ../DistUpgrade/DistUpgradeView.py:427
msgid "Reboot required"
msgstr "Herstart vereist"

#: ../DistUpgrade/DistUpgradeView.py:428
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Het opwaarderen is voltooid en herstarten is noodzakelijk. Wilt u dit nu "
"doen?"

#: ../DistUpgrade/DistUpgradeViewText.py:82
#, python-format
msgid "Progress: [%3i%%]"
msgstr "Voortgang: [%3i%%]"

#: ../DistUpgrade/DistUpgradeViewText.py:141
msgid ""
"Please report this as a bug and include the files /var/log/dist-upgrade/main."
"log and /var/log/dist-upgrade/apt.log in your report. The upgrade has "
"aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Rapporteer dit a.u.b. als een fout en voeg de bestanden /var/log/dist-"
"upgrade/main.log en /var/log/dist-upgrade/apt.log bij uw melding. De "
"opwaardering is afgebroken."

#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "Aborting"
msgstr "Afbreken"

#: ../DistUpgrade/DistUpgradeViewText.py:171
msgid "Demoted:\n"
msgstr "Gedegradeerd:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:178
msgid "To continue please press [ENTER]"
msgstr "Druk op [ENTER] om door te gaan"

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
msgid "Details [d]"
msgstr "Details [d]"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:222
#: ../DistUpgrade/DistUpgradeViewText.py:271
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:230
#, python-format
msgid "No longer supported: %s\n"
msgstr "Niet meer ondersteund: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:235
#, python-format
msgid "Remove: %s\n"
msgstr "Verwijderen: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:245
#, python-format
msgid "Install: %s\n"
msgstr "Installeren: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:250
#, python-format
msgid "Upgrade: %s\n"
msgstr "Opwaarderen: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:268
msgid "Continue [Yn] "
msgstr "Doorgaan [Jn] "

#: ../DistUpgrade/DistUpgradeViewText.py:293
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Om de opwaardering te voltooien moet u de computer herstarten.\n"
"Wanneer u 'j' selecteert zal de computer herstart worden."

#: ../DistUpgrade/GtkProgress.py:71
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Bestand %(current)li van %(total)li wordt gedownload met %(speed)s/s"

#: ../DistUpgrade/GtkProgress.py:77
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Bestand %(current)li van %(total)li wordt gedownload"

#: ../data/gtkbuilder/AcquireProgress.ui.h:1
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid "Show progress of individual files"
msgstr "Voortgang van de afzonderlijke pakketten tonen"

#: ../data/gtkbuilder/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "Opwaardering _annuleren"

#: ../data/gtkbuilder/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "_Opwaardering hervatten"

#: ../data/gtkbuilder/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>De actieve opwaardering annuleren?</big></b>\n"
"\n"
"Het systeem wordt mogelijk onbruikbaar als u de opwaardering annuleert. U "
"wordt sterk aangeraden om de opwaardering te hervatten."

#: ../data/gtkbuilder/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "Opwaardering _starten"

#: ../data/gtkbuilder/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "_Vervangen"

#: ../data/gtkbuilder/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Verschillen tussen de bestanden"

#: ../data/gtkbuilder/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "_Fout rapporteren"

#: ../data/gtkbuilder/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "_Doorgaan"

#: ../data/gtkbuilder/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Opwaardering starten?</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Herstart de computer om de opwaardering te voltooien</big></b>\n"
"\n"
"Sla a.u.b. uw werk op alvorens u verdergaat."

#: ../data/gtkbuilder/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "Distributie-opwaardering"

#: ../data/gtkbuilder/DistUpgrade.ui.h:20
#, no-c-format
msgid "<b><big>Upgrading Ubuntu to version %s</big></b>"
msgstr "<b><big>Ubuntu opwaarderen naar versie %s</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:21
msgid "    "
msgstr "    "

#: ../data/gtkbuilder/DistUpgrade.ui.h:23
msgid "Setting new software channels"
msgstr "Instellen van de nieuwe softwarekanalen"

#: ../data/gtkbuilder/DistUpgrade.ui.h:25
msgid "Restarting the computer"
msgstr "Computer herstarten"

#: ../data/gtkbuilder/DistUpgrade.ui.h:28
msgid "Terminal"
msgstr "Terminalvenster"

#: ../data/gtkbuilder/ReleaseNotes.ui.h:2
msgid "_Upgrade"
msgstr "_Opwaarderen"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid "<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr ""
"<b>Er is een nieuwe versie van Ubuntu beschikbaar. Wilt u opwaarderen?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Don't Upgrade"
msgstr "Niet opwaarderen"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Ask Me Later"
msgstr "Later opnieuw vragen"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Yes, Upgrade Now"
msgstr "Ja, nu opwaarderen"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "U heeft besloten om niet te op te waarderen naar de nieuwe Ubuntu"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""
"U kunt op een later tijdstip opwaarderen door Updatebeheer te openen en te "
"klikken op 'Opwaarderen'."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:1
msgid "Perform a release upgrade"
msgstr "Voer een versie-opwaardering uit"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:2
msgid "To upgrade Ubuntu, you need to authenticate."
msgstr "Voor een upgrade van Ubuntu is authenticatie vereist."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:3
msgid "Perform a partial upgrade"
msgstr "Een gedeeltelijke upgrade uitvoeren"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:4
msgid "To perform a partial upgrade, you need to authenticate."
msgstr "Voor een gedeeltelijke upgrade is authenticatie vereist."

#: ../do-partial-upgrade:75 ../do-release-upgrade:80
msgid "Show version and exit"
msgstr "Versie tonen en afsluiten"

#: ../do-partial-upgrade:78 ../do-release-upgrade:87
msgid "Directory that contains the data files"
msgstr "Map die de gegevensbestanden bevat"

#: ../do-partial-upgrade:81 ../do-release-upgrade:100
msgid "Run the specified frontend"
msgstr "De opgegeven 'frontend' uitvoeren"

#: ../do-partial-upgrade:112
msgid "Running partial upgrade"
msgstr "Gedeeltelijke opwaardering aan het uitvoeren"

#: ../do-release-upgrade:34 ../do-release-upgrade:46
msgid "Downloading the release upgrade tool"
msgstr "Gereedschap voor versie-opwaardering aan het binnenhalen"

#: ../do-release-upgrade:83
msgid ""
"If using the latest supported release, upgrade to the development release"
msgstr ""
"Indien de nieuwste ondersteunde versie gebruikt wordt, waardeer op naar de "
"ontwikkelversie"

#: ../do-release-upgrade:90 ../check_new_release_gtk.py:186
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Probeer op te waarderen naar de nieuwste versie door gebruik te maken van "
"het opwaardeergereedschap van $distro-proposed"

#: ../do-release-upgrade:94
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Uitvoeren in een speciale opwaardeermodus\n"
"Momenteel worden 'desktop' en 'server' ondersteund voor opwaarderen van "
"respectievelijk een bureaubladsysteem en een serversysteem."

#: ../do-release-upgrade:103
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Alleen controleren wanneer er een nieuwe distributie-uitgave beschikbaar is, "
"en het resultaat rapporteren via de afsluitstatus"

#: ../do-release-upgrade:108
msgid ""
"Try the upgrade with third party mirrors and repositories enabled instead of "
"commenting them out."
msgstr ""
"Probeer te upgraden met mirrors en softwarebronnen van derden ingeschakeld, "
"in plaats van deze uit te sluiten."

#: ../do-release-upgrade:122
msgid "The options --devel-release and --proposed are"
msgstr "De opties --devel-release en --proposed zijn"

#: ../do-release-upgrade:123
msgid "mutually exclusive. Please use only one of them."
msgstr "niet samen te gebruiken. Gebruik slechts een van de twee."

#: ../do-release-upgrade:127
msgid "Checking for a new Ubuntu release"
msgstr "Er wordt gecontroleerd of er een nieuwe Ubuntu-uitgave is"

#: ../do-release-upgrade:139
msgid "Your Ubuntu release is not supported anymore."
msgstr "Uw Ubuntu-versie wordt niet langer ondersteund."

#: ../do-release-upgrade:140
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Voor opwaardeerinformatie, bezoek:\n"
"%(url)s\n"

#: ../do-release-upgrade:155
msgid "In /etc/update-manager/release-upgrades Prompt "
msgstr "In /etc/update-manager/release-upgrades is de prompt "

#: ../do-release-upgrade:156
msgid "is set to never so upgrading is not possible."
msgstr "ingesteld op nooit, dus is upgraden niet mogelijk."

#: ../do-release-upgrade:158
msgid "There is no development version of an LTS available."
msgstr "Er is geen ontwikkelversie van een LTS beschikbaar."

#: ../do-release-upgrade:159
msgid "To upgrade to the latest non-LTS development release "
msgstr ""

#: ../do-release-upgrade:160
msgid "set Prompt=normal in /etc/update-manager/release-upgrades."
msgstr "geef Prompt=normal in /etc/update-manager/release-upgrades op."

#: ../do-release-upgrade:162
msgid "Upgrades to the development release are only "
msgstr "Opwaarderingen naar de ontwikkelversie zijn alleen "

#: ../do-release-upgrade:163
msgid "available from the latest supported release."
msgstr "beschikbaar in de nieuwste ondersteunde versie."

#: ../do-release-upgrade:165
msgid "No new release found."
msgstr "Geen nieuwe versie gevonden."

#: ../do-release-upgrade:170
msgid "Release upgrade not possible right now"
msgstr "Uitgave-opwaardering momenteel niet mogelijk"

#: ../do-release-upgrade:171
#, c-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"De uitgave-opwaardering kan momenteel niet plaatsvinden, probeer het a.u.b. "
"later nog eens. De server gaf als antwoord: ‘%s’"

#: ../do-release-upgrade:177
#, c-format
msgid "New release '%s' available."
msgstr "Nieuwe versie '%s' beschikbaar."

#: ../do-release-upgrade:178
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr ""
"Voer 'do-release-upgrade' uit om naar de nieuwe versie op te waarderen."

#: ../do-release-upgrade:195
msgid "Please install all available updates for your release before upgrading."
msgstr ""
"Gelieve eerst alle beschikbare updates voor uw versie te installeren "
"alvorens u begint met de upgrade."

#: ../do-release-upgrade:211
msgid ""
"You have not rebooted after updating a package which requires a reboot. "
"Please reboot before upgrading."
msgstr ""
"U bent niet opnieuw opgestart na het bijwerken van een pakket waarvoor "
"opnieuw opstarten vereist is. Start opnieuw op voordat u een upgrade "
"uitvoert."

#: ../check_new_release_gtk.py:86
msgid "Software Updater"
msgstr "Updatebeheer"

#: ../check_new_release_gtk.py:118
#, python-format
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Opwaardering naar Ubuntu %(version)s beschikbaar"

#. print("don't upgrade")
#: ../check_new_release_gtk.py:149
#, python-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "U heeft besloten om niet te op te waarderen naar Ubuntu %s"

#: ../check_new_release_gtk.py:182
msgid "Check if upgrading to the latest devel release is possible"
msgstr "Controleer of opwaarderen naar de nieuwste ontwikkelversie mogelijk is"

#: ../check_new_release_gtk.py:191
msgid "Add debug output"
msgstr "Debug-uitvoer toevoegen"

#~ msgid "Could not calculate sources.list entry"
#~ msgstr "Berekenen van sources.list-vermelding is mislukt"

#~ msgid ""
#~ "Unable to locate any package files, perhaps this is not a Ubuntu Disc or "
#~ "the wrong architecture?"
#~ msgstr ""
#~ "Geen pakketbestanden gevonden; mogelijk is dit geen Ubuntu-medium of "
#~ "beschikt u niet over de juiste hardware-architectuur."

#~ msgid "Failed to add the CD"
#~ msgstr "Toevoegen van de cd is mislukt"

#~ msgid ""
#~ "There was a error adding the CD, the upgrade will abort. Please report "
#~ "this as a bug if this is a valid Ubuntu CD.\n"
#~ "\n"
#~ "The error message was:\n"
#~ "'%s'"
#~ msgstr ""
#~ "Er is een fout opgetreden tijdens het toevoegen van de cd, waardoor de "
#~ "opwaardering onderbroken zal worden. Gelieve deze fout te rapporteren "
#~ "indien dit een geldige Ubuntu-cd is.\n"
#~ "\n"
#~ "De foutmelding was:\n"
#~ "‘%s’"

#~ msgid "Include latest updates from the Internet?"
#~ msgstr "Laatste updates van het internet ophalen?"

#~ msgid ""
#~ "The upgrade system can use the internet to automatically download the "
#~ "latest updates and install them during the upgrade.  If you have a "
#~ "network connection this is highly recommended.\n"
#~ "\n"
#~ "The upgrade will take longer, but when it is complete, your system will "
#~ "be fully up to date.  You can choose not to do this, but you should "
#~ "install the latest updates soon after upgrading.\n"
#~ "If you answer 'no' here, the network is not used at all."
#~ msgstr ""
#~ "Bij de opwaardering kunnen automatisch de laatste updates van het "
#~ "internet opgehaald en geïnstalleerd worden. Als u een netwerkverbinding "
#~ "heeft is dit ten zeerste aan te bevelen.\n"
#~ "\n"
#~ "Het opwaardeerproces zal hierdoor langer duren, maar na afloop is uw "
#~ "systeem geheel bijgewerkt. Als u hier niet voor kiest zult u snel na de "
#~ "opwaardering de updates moeten installeren.\n"
#~ "Als u voor 'nee' kiest, wordt het netwerk helemaal niet gebruikt."

#~ msgid "Failed to add the cdrom"
#~ msgstr "Toevoegen van cdrom mislukt"

#~ msgid "Sorry, adding the cdrom was not successful."
#~ msgstr "Helaas is het niet gelukt de cdrom toe te voegen"

#~ msgid "Use the given path to search for a cdrom with upgradable packages"
#~ msgstr ""
#~ "Gebruik het opgegeven pad om te zoeken naar een cd-rom met pakketten die "
#~ "geüpgraded kunnen worden"

#~ msgid ""
#~ "Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
#~ "desktop or edubuntu-desktop package and it was not possible to detect "
#~ "which version of Ubuntu you are running.\n"
#~ " Please install one of the packages above first using synaptic or apt-get "
#~ "before proceeding."
#~ msgstr ""
#~ "Uw systeem bevat geen ubuntu-desktop-, kubuntu-desktop-, xubuntu-desktop- "
#~ "of edubuntu-desktop-pakket waardoor het niet mogelijk was om te bepalen "
#~ "welke variant van Ubuntu u heeft.\n"
#~ " Installeer eerst één van de bovenstaande pakketten met synaptic of apt-"
#~ "get voordat u verder gaat."

#~ msgid "Sandbox setup failed"
#~ msgstr "Sandbox-installatie mislukt"

#~ msgid "Sandbox mode"
#~ msgstr "Sandbox-modus"

#~ msgid ""
#~ "Your python install is corrupted. Please fix the '/usr/bin/python' "
#~ "symlink."
#~ msgstr ""
#~ "Uw python-installatie is beschadigd. Herstel de symbolische link ‘/usr/"
#~ "bin/python’."

#~ msgid "Package 'debsig-verify' is installed"
#~ msgstr "Pakket 'debsig-verify' is geïnstalleerd."

#~ msgid "evms in use"
#~ msgstr "evms wordt gebruikt"

#~ msgid ""
#~ "This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
#~ "version of this driver is available that works with your video card in "
#~ "Ubuntu 10.04 LTS.\n"
#~ "\n"
#~ "Do you want to continue?"
#~ msgstr ""
#~ "Deze computer gebruikt momenteel het grafisch stuurprogramma 'nvidia' van "
#~ "NVIDIA. Voor Ubuntu 10.04 LTS is geen stuurprogramma beschikbaar dat met "
#~ "uw hardware werkt.\n"
#~ "Wilt u doorgaan?"

#~ msgid "No i686 CPU"
#~ msgstr "Geen i686-CPU"

#~ msgid "No new release found"
#~ msgstr "Geen nieuwe versie gevonden"

#~ msgid ""
#~ "If none of this applies, then please report this bug using the command "
#~ "'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
#~ msgstr ""
#~ "Indien niets van dit alles van toepassing is, meld deze fout dan a.u.b. "
#~ "met behulp van de terminalopdracht 'ubuntu-bug ubuntu-release-upgrader-"
#~ "core'."

#~ msgid "Authentication is required to perform a partial upgrade"
#~ msgstr ""
#~ "Authenticatie is vereist om een gedeeltelijke opwaardering uit te voeren"

#~ msgid "Authentication is required to perform a release upgrade"
#~ msgstr "Authenticatie is vereist om een versie-opwaardering uit te voeren"

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade:\n"
#~ "%s\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "Er deed zich een onoplosbaar probleem voor tijdens het berekenen van de "
#~ "opwaardering:\n"
#~ "%s\n"
#~ "\n"
#~ " Dit kan veroorzaakt worden door:\n"
#~ " * het opwaarderen naar een nog niet uitgebrachte versie van Ubuntu\n"
#~ " * het gebruik van de huidige niet-uitgebrachte versie van Ubuntu\n"
#~ " * onofficiële programma's, niet geleverd door Ubuntu\n"
#~ "\n"

#~ msgid ""
#~ "While scanning your repository information no mirror entry for the "
#~ "upgrade was found. This can happen if you run a internal mirror or if the "
#~ "mirror information is out of date.\n"
#~ "\n"
#~ "Do you want to rewrite your 'sources.list' file anyway? If you choose "
#~ "'Yes' here it will update all '%s' to '%s' entries.\n"
#~ "If you select 'No' the upgrade will cancel."
#~ msgstr ""
#~ "Tijdens het doorzoeken van uw beschikbare softwarebronnen is er geen "
#~ "spiegelserver voor de opwaardering gevonden. Dit kan gebeuren als u een "
#~ "interne spiegelserver heeft of als de spiegelserverinformatie verouderd "
#~ "is.\n"
#~ "\n"
#~ "Wilt u ‘sources.list’ toch herschrijven? Als u 'Ja' kiest, worden "
#~ "spiegelservers ‘%s’ tot ‘%s’ bijgewerkt. Bij ‘Nee’ wordt de opwaardering "
#~ "geannuleerd."

#~ msgid ""
#~ "The upgrade can not continue with that package installed.\n"
#~ "Please remove it with synaptic or 'apt-get remove debsig-verify' first "
#~ "and run the upgrade again."
#~ msgstr ""
#~ "De opwaardering kan niet worden voortgezet als dat pakket geïnstalleerd "
#~ "is.\n"
#~ "Verwijder het eerst met synaptic of 'apt-get remove debsig-verify' en "
#~ "voer de opwaardering opnieuw uit."

#~ msgid ""
#~ "The upgrade has aborted. The upgrade needs a total of %s free space on "
#~ "disk '%s'. Please free at least an additional %s of disk space on '%s'. "
#~ "Empty your trash and remove temporary packages of former installations "
#~ "using 'sudo apt-get clean'."
#~ msgstr ""
#~ "Het opwaarderen is afgebroken. De installatie heeft in totaal %s vrije "
#~ "schijfruimte nodig op schijf ‘%s’. Maak tenminste een extra %s "
#~ "schijfruimte vrij op ‘%s’. Maak de prullenbak leeg en verwijder "
#~ "tijdelijke bestanden van vorige installaties via de opdracht 'sudo apt-"
#~ "get clean'."

#~ msgid ""
#~ "Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
#~ "software is no longer supported, please switch it off and run the upgrade "
#~ "again when this is done."
#~ msgstr ""
#~ "Uw systeem gebruikt de volumebeheerder 'evms' in /proc/mounts. De 'evms'-"
#~ "programmatuur wordt niet langer ondersteund, schakel hem a.u.b. uit en "
#~ "voer de opwaardering nogmaals uit wanneer dit gebeurd is."

#~ msgid ""
#~ "Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
#~ "extension. All packages were built with optimizations requiring i686 as "
#~ "the minimal architecture. It is not possible to upgrade your system to a "
#~ "new Ubuntu release with this hardware."
#~ msgstr ""
#~ "Uw systeem gebruikt een i586-CPU of een CPU die geen ‘cmov’-extensie "
#~ "heeft. Alle pakketten zijn gemaakt met optimalisaties die minstens i686 "
#~ "nodig hebben. Met deze apparatuur is het niet mogelijk om uw systeem op "
#~ "te waarderen naar een nieuwe versie van Ubuntu."

#~ msgid "Sandbox upgrade using aufs"
#~ msgstr "Sandbox-opwaardering via aufs"

#~ msgid "Test upgrade with a sandbox aufs overlay"
#~ msgstr "Opwaardering uitproberen met een sandbox-aufs-overlay"

#~ msgid "It was not possible to create the sandbox environment."
#~ msgstr "Het aanmaken van de sandbox-omgeving was niet mogelijk."

#~ msgid ""
#~ "This upgrade is running in sandbox (test) mode. All changes are written "
#~ "to '%s' and will be lost on the next reboot.\n"
#~ "\n"
#~ "*No* changes written to a system directory from now until the next reboot "
#~ "are permanent."
#~ msgstr ""
#~ "Deze opwaardering wordt in sandbox (test)-modus uitgevoerd. Alle "
#~ "wijzigingen worden naar ‘%s’ weggeschreven en zullen verloren gaan als u "
#~ "de computer afsluit.\n"
#~ "\n"
#~ "*Geen enkele* wijziging aan een systeemmap vanaf nu totdat u de computer "
#~ "opnieuw opstart zal permanent zijn."

#~ msgid "<b><big>Upgrading Ubuntu to version 15.04</big></b>"
#~ msgstr "<b><big>Ubuntu opwaarderen naar versie 15.04</big></b>"

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade.\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "Er is een onoplosbaar probleem geconstateerd bij het berekenen van de "
#~ "opwaardering.\n"
#~ "\n"
#~ " De oorzaak hiervan kan zijn:\n"
#~ " * Opwaardering naar een voorlopige versie van Ubuntu\n"
#~ " * Uitvoering van de huidige voorlopige versie van Ubuntu\n"
#~ " * Onofficiële softwarepakketten die niet door Ubuntu geleverd zijn.\n"
#~ "\n"

#~ msgid "<b><big>Upgrading Ubuntu to version 15.10</big></b>"
#~ msgstr "<b><big>Ubuntu opwaarderen naar versie 15.10</big></b>"
